rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -0,0 +1,199 @@
+The DCP IRT follows the incident response lifecycle described below.
+
+![Incident Response Lifecycle](https://drive.google.com/uc?export=view&id=1cTXM31emPeS2RiaSI6p8yLfyZG8a6_R3)
+
+##### Phase 1: Preparation
+Preventing incidents from occurring is a part of preparation. DCP personnel implement appropriate security controls as part of the overall DevSecOps program. Preparation activities related to incident response are as follows:
+* Establish an incident response capability
+* Develop incident response policies and procedures
+
+##### Phase 2: Detection and Analysis
+Incident identification occurs in this phase. The DCP IRT learns of an actual or suspected security incident. Incidents may be detected through many different means with varying levels of detail and veracity. Security incidents may be detected through some of the following examples:
+* End user identification
+* System administrator identification
+* Log review
+* Automated alerting tools
+* Locked out accounts
+* Pop-ups and redirected websites when browsing 
+* Unexpected software installs
+* Unexplained changes to files
+* Anomalies in normal network traffic patterns
+* Abnormal outbound traffic
+* Irregular access locations
+* Large number of requests for the same object or files
+* Suspicious activity on the network after-hours
+* Multiple failed login attempts
+* Unknown/unauthorized IP addresses
+* Services and applications configured to launch automatically
+
+If there is confirmation of an incident, the IRT formulates an overall response strategy and reports the incident to the Oversight Committee and other concerned parties, as appropriate.
+
+##### Phase 3: Containment, Eradication, and Recovery
+This phase is used to gather electronic and physical evidence to determine exactly what has happened, who might be responsible, and how this type of intrusion can be prevented in the future.  The DCP IRT will make recommendations to remediate affected resources and verify completion. As necessary, the IRT will take measures to limit the amount of damage that an intruder or authorized affected system can inflict upon the rest of the IT infrastructure. 
+
+##### Phase 4: Post-Incident Activity
+Follow-up occurs in this phase. DCP IRT completes a report of the incident, subsequent recovery process, and lessons learned. A final copy of the incident report is sent to the Oversight Committee and appropriate stakeholders. Additionally, an incident post-mortem may be conducted to evaluate the lessons learned during the event.
+
+#### Incident Response Procedures
+Anyone can declare an incident. Follow the procedure outlined below if you suspect or have confirmed that an incident has occurred.
+
+__Step 1__: Personnel identifying the suspected or confirmed incident (Incident Handler) will alert the DCP IRT by submitting a ticket to the to-be-created humancellatlas/dcp-security private GitHub repo. The ticket will include the following information to ensure appropriate triage:
```

Comment:
Will this trigger PagerDuty or another alerting tool for high severity incidents?

Reply:
No; this IRP doesn't include Security Operations Center (SOC) level response. We may one day require that, but it will be a future enhancement to process as needed.